crates/telio-model/src/features.rs

```diff
@@ -568,22 +586,32 @@ pub struct Features {
     pub multicast: bool,
 }
 
-impl FeaturePaths {
-    /// Returns a vector of 'PathType' sorted according to the priority
-    pub fn paths(&self) -> Vec<PathType> {
-        if let Some(path) = self.force {
-            vec![path]
-        } else {
-            // Collect PathType array, without any duplicates, and keeping prio for priorities
-            [PathType::Relay]
-                .iter()
-                .chain(self.priority.iter())
-                .fold(Vec::new(), |mut v, p| {
-                    if !v.contains(p) {
-                        v.push(*p);
-                    }
-                    v
-                })
+impl Features {
+    fn default_nurse() -> Option<FeatureNurse> {
+        Some(Default::default())
+    }
+}
+
+impl Default for Features {
+    fn default() -> Self {
```

Comment:
Unfortunately we now have to implement default like this because using `..Default::default()` would make it a recursive implementation